src/rest_framework_dso/exceptions.py

```diff
@@ -42,3 +42,17 @@ def __init__(self, title, detail, code, status_code):
         self.code = code or self.default_code
         self.default_detail = title
         self.status_code = status_code
+
+
+class HumanReadableException(APIException):
+    """Base class for tagging exceptions that can be shown in complete detail to the client.
+    This can be shown in the standard exception handler, or during streaming rendering.
+    """
+
+    default_detail = "Error During Rendering"
```

Comment:
```suggestion
    default_detail = "error during rendering"
```

Reply:
I've used uppercase-first instead of title-case, which is indeed in line with other APIException classes from DRF.